ng-jcrop.js

```diff
@@ -166,6 +167,10 @@
         $scope.getShrinkRatio = function(){
             var img = $('<img>').attr('src', $scope.mainImg[0].src)[0];
 
+            if(ngJcropConfig.jcrop.maxWidth > img.width || ngJcropConfig.jcrop.maxHeight > img.height){
```

Comment:
You're right. Can I ask you a favor? Add examples in this tests : https://github.com/andrefarzat/ng-jcrop/blob/master/test/ng-jcrop_test.js#L248
So we can fix it better =)